### PR TITLE
DSD-1795: Breadcrumbs Digital Collections variant

### PR DESCRIPTION
**Pull request by 7emansell**
Fixes JIRA ticket [DSD-1795](https://newyorkpubliclibrary.atlassian.net/browse/DSD-1795).

## This PR does the following:

- Adds the `digitalCollections` color variant for `Breadcrumbs` component

### Checklist:

- [x] I have updated the Storybook documentation accordingly.
- [x] I have added relevant accessibility documentation for this pull request.
- [x] All new and existing tests passed.

### Front End Review:

- [x] Review the Vercel preview deployment once it is ready.

[DSD-1795]: https://newyorkpubliclibrary.atlassian.net/browse/DSD-1795?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by EdwinGuzman — CHANGES_REQUESTED** [review]
This looks good but look at the `blogs` variant and the `customFocusColor ` function and how it handles its focus outline color. It's currently white on a light background and it won't pass accessibility reviews:
Screenshot 2024-06-26 at 10 10 16 AM

File: src/components/Breadcrumbs/Breadcrumbs.tsx [hunk 1]
```diff
@@ -21,6 +21,7 @@ export const breadcrumbTypeArray = [
   "locations",
   "research",
   "whatsOn",
+  "digitalCollections",
```
Comment (bigfishdesign13) [inline src/components/Breadcrumbs/Breadcrumbs.tsx]: Alphabetize this option with the others.

File: src/components/Breadcrumbs/Breadcrumbs.stories.tsx [hunk 2]
```diff
@@ -132,6 +132,15 @@ export const ColorVariations: Story = {
...
       </Box>
+      <Box>
+        <Heading level="h3" size="heading6">
+          Digital Collections
+        </Heading>
+        <Breadcrumbs
+          breadcrumbsData={breadcrumbsData}
+          breadcrumbsType="digitalCollections"
+        />
+      </Box>
```
Comment (bigfishdesign13) [inline src/components/Breadcrumbs/Breadcrumbs.stories.tsx]: To keep the full list alphabetized. move this example between `Connect` and `Education`.

File: src/theme/components/breadcrumb.ts [hunk 3]
```diff
@@ -74,6 +74,35 @@ const whatsOn = defineStyle({
     bg: "dark.section.whats-on.secondary",
   },
 });
+const digitalCollections = defineStyle({
```
Comment (bigfishdesign13) [inline src/theme/components/breadcrumb.ts:53]: Edwin already mentioned this, but take a look at how `customFocusColor` is used for the `blogs` variant.  The light background color necessitates a dark forcus ring.

**Review by bigfishdesign13 — CHANGES_REQUESTED** [review]
Looking good.  This just needs a couple of adjustments.

**Review by EdwinGuzman — APPROVED** [review]

File: src/components/Breadcrumbs/Breadcrumbs.tsx (hunk 1 above)
Comment (bigfishdesign13) [reply src/components/Breadcrumbs/Breadcrumbs.tsx]: This still needs to be alphabetized in the array.

**Review by bigfishdesign13 — APPROVED** [review]
Looks good.  Just alphebatize the items in the array before you merge.

File: src/components/Breadcrumbs/Breadcrumbs.tsx (hunk 1 above)
Comment (7emansell) [reply src/components/Breadcrumbs/Breadcrumbs.tsx]: Oops thank you!!

**Pull request merged**